docs/finality-provider-operation.md

```diff
@@ -373,6 +376,27 @@ this value by specifying a custom address with the `--rpc-listener` flag.
 EOTS Manager Daemon is fully active!
 ```
 
+#### 5.3.1. Unlock file-based keyring
+
+⚠️ If you are using a `file` based keyring-backend, you need to unlock the keyring by executing the following command:
+
+```shell
+eotsd unlock --eots-pk <eots-pk> --rpc-client <eotsd-address>
+```
+
+You will be prompted to enter the password for the keyring. After which the signing operations will be available and eotsd can
+run uninterrupted.
+
+Alternatively to inputting the password, you can specify the password with the environment variable `EOTS_KEYRING_PASSWORD`:
+
+```shell
+export EOTSD_KEYRING_PASSWORD=<your-password>
+```
+
+If you have HMAC security enabled, you can also specify the HMAC key either with:
+- `HMAC_KEY` environment variable
+- Providing the `--home` path to the eotsd home directory which contains the config file.
+
 >**🔒 Security Tip**:
```

Comment:
The security tip seems to be for the 5.3.1. section, but it's a general one, I suggest we move it before the section as before.